cmd/cmd.go

```diff
@@ -66,17 +72,56 @@ func CreateHandler(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	var spinner *Spinner
+	modelfile, err := os.Open(filename)
```

Comment:
I don't think we need to create another abstraction here. I think we can just pass the modelfile itself. It's OK to validate that the modelfile is correct before sending it, but the commands abstraction makes it difficult for other clients to call the API call successfully.